Sketchfab-Exporter.pyp

```diff
@@ -15,22 +15,17 @@ Modified Date: 03/16/13
 import c4d
 from c4d import documents, gui, plugins, bitmaps, storage
 
-import time
 import datetime
-import sys
 import os
 import json
-import base64
 import urllib2
 import shelve
-import logging
 import webbrowser
 import zipfile
 import threading
 
 
-
-#logging.basicConfig(level=logging.DEBUG)
+# logging.basicConfig(level=logging.DEBUG)
```

Comment:
I'd rather remove all commented code than fixing them to be pep8 compliant. That's what git is for